libs/api/domains/work-machines/src/lib/api-domains-work-machines.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common'
+import { WorkMachinesResolver } from './api-domains-work-machines.resolver'
+import { WorkMachinesService } from './api-domains-work-machines.service'
+import { WorkMachinesClientModule } from '@island.is/clients/work-machines'
+import { FeatureFlagModule } from '@island.is/nest/feature-flags'
+
+@Module({
+  imports: [WorkMachinesClientModule, FeatureFlagModule],
+  providers: [WorkMachinesResolver, WorkMachinesService],
+})
+export class WorkMachinesModule {}
```

Comment:
The modeler in me dreams of this living in the vehicle domain and ideally using the same object types. Since our graphql schema is becoming more and more bloated 😅

Can understand that it's way more work and not possible in the time frame given. But maybe something to think about in the future 👍

Reply:
That's a good shout, it'll go on the docket